docs/solutions/sql_server_always_on_availability_groups/modules/network/securitylist/main.tf

```diff
@@ -0,0 +1,430 @@
+resource "oci_core_security_list" "dmz" {
+  compartment_id = "${var.compartment_id}"
+  display_name   = "DMZ.sl"
+  vcn_id         = "${var.vcn_id}"
+
+  egress_security_rules = [
+    {
+      protocol    = "${var.proto_number_All}"
+      destination = "0.0.0.0/0"
+    },
+  ]
+
+  ingress_security_rules = [
+    {
+      tcp_options {
+        "max" = 3389
```

Comment:
Would be good to have some comments in this file explaining where these rules came from.